### ISSUE TITLE
Decide between Drone.io or Strider-CD for Continuous Integration

### ISSUE THREAD

**Issue by chasinglogic** [specz187/chasinglogic-server-configs#17]
Need to solidify our choice of stack then make sure we get it set up properly since our drone install is wonky and having problems. The important bit here is not just being able to build / test our code but most importantly being able to **deploy** our code. Test each solution and decide whether troubleshooting the Drone instance or jumping ship to Strider is the right call. Note that due to possible Node version conflicts Strider will have to be installed using a docker image which we have to create as they do not provide an official image. 

[Here are the instructions for installing Strider on Ubuntu](https://fosterelli.co/creating-a-private-ci-with-strider.html)

Follow those inside a docker image and then commit that image as Strider:chasinglogic.com 

[The Docker documentation is here](http://docs.docker.com/linux/started/)

**Comment by chasinglogic** [comment]
Both solutions will make heavy use of Docker so I recommend reading the linked Docker docs either way.

**Comment by chasinglogic** [comment]
I configured Drone I have it set up on a seperate server now and use nginx to proxy to it. Currently works pretty well and I created a chasinglogic:build image for use with it. Got most of the nasty GOPATH stuff figured out though it certainly does feelbad.jpg but that's what we get when using an alpha. Strider CD was not what I wanted and Gitlab CI required build agents which I also did not want to do.
